mail_connector_queue/readme/USAGE.rst

```diff
@@ -0,0 +1,8 @@
+A queue job is created automatically when a new ``mail.mail`` record is
+created in the ``outgoing`` state. A priority may be defined in the
+``mail.mail`` record, which will be used as the job priority.
+
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/102/8.0
```

Comment:
@sbidoul Should I commit the generated README?

Reply:
Normally it's not necessary. But since the README generation system is new, it could be nice so we can have an visual check. Make sure to use the latest master branch of OCA/maintainer-tools

Reply:
@sbidoul If I understood well, no README.rst file is needed at all with the new system. Only the fragments. Is that true? We are trying the new system also in https://github.com/OCA/website/pull/464. Can you check for pointing us anything relevant?